CHANGELOG.adoc

```diff
@@ -18,12 +31,18 @@ toc::[]
 
 === Changed
 
+=== Deprecated
```

Comment:
This is doubled and needs to be removed as the right position is line 44

Reply:
I removed the redundant header - however I sticked to latest change made by you in DD, so which is correct then line 34 or 44?